### DISCUSSION TITLE
网易云游戏 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#18] [模板讨论]
网易云游戏 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by gitcps** [comment]
首页cookie抓取没用

**Reply by wjf0214** [reply]
我观察了几天。
看网页显示的签到情况，是正常有在签到的。模板没问题。
如果添加的时候报错了。可能是已经签到过了。当天签到后，再次执行就会报错。

**Comment by miaoxiaozhao** [comment]
这个Authorization是不是只能在APP里面抓取？我印象里很早以前网页可以找到的，但是现在忘记在哪里了

**Reply by wjf0214** [reply]
我看了下，网页是有的。你点开链接看看，很多请求里的请求标头里都有。

**Reply by floralhoop** [reply]
请问需不需要带 Bearer?

**Reply by floralhoop** [reply]
![image](https://github.com/wjf0214/qd-templates/assets/78347632/e0929252-a070-4c30-84ad-ad4a4648de25)

**Reply by wjf0214** [reply]
需要带，可以试试。执行完看看网页有没有完成签到就行了。

**Reply by floralhoop** [reply]
![image](https://github.com/wjf0214/qd-templates/assets/78347632/56b339cf-fac7-45e9-b6f8-bd9c9b4b1379)
一直报这个  死活签到不了

**Reply by floralhoop** [reply]
是不是手动签到后这个不会有正常返回?

**Reply by wjf0214** [reply]
你已经签到过了吧？签到过了是这样。一般没问题，明天再看看就是了。

**Reply by floralhoop** [reply]
好的好的 谢谢大佬贡献的优秀模板
